test/bare.js

```diff
@@ -68,3 +75,72 @@ test('bare inserts __filename,__dirname but not process,global,Buffer', function
         t.equal(code, 0);
     });
 });
+
+test('bare inserts dynamic __filename,__dirname', function (t) {
```

Comment:
These tests execute the following file: https://github.com/substack/node-browserify/pull/1725/files#diff-df4230a729c19a353d1405d5459df60d, which prints both `__dirname`, and `__filename` (and we assert the result of both)